ONBOARDING.md

```diff
@@ -3,67 +3,34 @@
-* Release CI (ci-release.nodejs.org)
-  * `nodejs/jenkins-release-admins` are administrators
-  * What creates and publishes all releases of Node.js
-
-### Secrets
-
-* All production secrets are stored in the `build` directory of the
-[`nodejs-private/secrets` repository][]
-* The access groups are explained in [`doc/access.md`](doc/access.md)
-* Say you want to decrypt the file `a.txt`, you'd run `dotgpg cat a.txt`
-* Follow the instructions in [`doc/ssh.md`](doc/ssh.md) to setup the secrets
-  repo and SSH access locally
-
-### Ansible
-
-* Ansible scripts are used to setup and maintain all machines
-* See [`doc/services.md`](doc/services.md) for which playbooks
-  correspond to different machines
-* Go over [`ansible/README.md`](ansible/README.md) for Ansible setup
-  instructions
-* Have the onboardee practice running the `jenkins/worker/create.yml` playbook
-  on one of the machines in the test CI cluster
-* Windows access
-
-### Communication
-
-* GitHub Issues are used to manage tasks within the Build WG
-* Use GitHub Pull Requests to suggest, and then land, changes
-  * Keep open for at least 48 hours during the week (72 hours on the weekend)
-  * Get at least one approval from another WG member
-  * Land PRs with the same commit metadata you'd use in `nodejs/node`
-* IRC (specifically `#node-build` is important for communicating with
-  other Node.js project members, and how we receive many initial signals
-  of downtime
-    * IRC logs are maintained at http://logs.libuv.org/node-build
-* Build WG meetings are every 2-3 weeks, and you should try and attend
-  as many as possible. The meetings are listed in the [Node.js
-Foundation calendar][].
-
-[`nodejs-private/secrets` repository]: https://github.com/nodejs-private/secrets
-[Node.js Foundation calendar]: https://nodejs.org/calendar
+## Onboarding checklists
+
+### Onboarding to build-test issue checklist
+
+- [ ] Email the new member and obtain their public GPG key
+- [ ] Add them to the `test` group in `nodejs-private/secrets`
+- [ ] Add them to the `nodejs/build` and `nodejs-private/build` GitHub teams
+- [ ] Before the meeting the Onboardee is to:
+  - [ ] Install Ruby and the `dotgpg` gem
+  - [ ] Install Python and Ansible
+  - [ ] Read `services.md`. `jenkins-guide.md`,
+    `GOVERNANCE.md` and have any questions ready
+- [ ] Schedule a meeting with the member to:
+    - [ ] Walk them through the infrastructure and what other members do
+    - [ ] Explain how to decrypt nodejs/secrets 
+    - [ ] Practice running the `jenkins/worker/create.yml` playbook on one of the machines in the test CI cluster
+    - [ ] Answer any questions they may have
+- [ ] PR changes to [README.md](./README.md#build-wg-members) to add the member to build-test
+
+
+At first all members are added to nodejs/build-test. As their contribution and experience grows
+within the working group they may be granted further permissions and access. Examples of further access include:
```

Comment:
I suggest removing from "Examples of.." right to the end. They are all examples, but this is better discussed in access.md, its not part of the the onboarding checklist. Actually, everything from line 26 above down (the stuff after the checklist) I think could just be deleted without being missed.